rules/falco_rules.yaml

```diff
@@ -746,6 +746,13 @@
 - macro: runuser_reading_pam
   condition: (proc.name=runuser and fd.directory=/etc/pam.d)
 
+# CIS Linux Benchmark program
+- macro: linux_bench_reading_etc_shadow
+  condition: ((proc.aname[2]=linux-bench and
```

Comment:
can we use `proc.aname=linux-bench` instead of specifying the index. linux-bench doesn't need to be the grant parent of awk, cut, grep

Reply:
proc.aname is a bit more expensive, as it traverses the history until it finds an ancestor with the name or runs into init/the root process. We could do that to make it more flexible, but the cost will be higher for the negative case.